tests/functional/aws-node-sdk/test/object/get.js

```diff
@@ -57,36 +59,79 @@ describe('GET object', () => {
-                s3.putObject(params, err => done(err));
+            describe('if specified in put object request', () => {
+                before(done => {
+                    const params = {
+                        Bucket: bucketName,
+                        Key: objectName,
+                        CacheControl: cacheControl,
+                        ContentDisposition: contentDisposition,
+                        ContentEncoding: contentEncoding,
+                        ContentType: contentType,
+                        Expires: expires,
+                    };
+                    s3.putObject(params, err => done(err));
+                });
+                it('should return additional headers', done => {
+                    s3.getObject({ Bucket: bucketName, Key: objectName },
+                      (err, res) => {
+                          if (err) {
+                              return done(err);
+                          }
+                          assert.strictEqual(res.CacheControl,
+                            cacheControl);
+                          assert.strictEqual(res.ContentDisposition,
+                            contentDisposition);
+                          // Should remove V4 streaming value 'aws-chunked'
+                          // to be compatible with AWS behavior
+                          assert.strictEqual(res.ContentEncoding,
+                            'gzip');
+                          assert.strictEqual(res.ContentType, contentType);
+                          assert.strictEqual(res.Expires,
+                              expires.toGMTString());
+                          return done();
+                      });
+                });
             });
-            it('should return additional headers if specified in objectPUT ' +
-                'request', done => {
-                s3.getObject({ Bucket: bucketName, Key: objectName },
-                  (err, res) => {
-                      if (err) {
-                          return done(err);
-                      }
-                      assert.strictEqual(res.CacheControl,
-                        cacheControl);
-                      assert.strictEqual(res.ContentDisposition,
-                        contentDisposition);
-                      // Should remove V4 streaming value 'aws-chunked'
-                      // to be compatible with AWS behavior
-                      assert.strictEqual(res.ContentEncoding,
-                        'gzip');
-                      assert.strictEqual(res.Expires,
-                          expires.toGMTString());
-                      return done();
-                  });
+
+            describe('if response content headers are set in query', () => {
+                before(done => {
+                    s3.putObject({ Bucket: bucketName, Key: objectName },
+                        err => done(err));
+                });
+
+                it('should return additional headers even if not set in ' +
```

Comment:
yep -- but they are set in the objectGet request, not the objectPut request (see next line), in contrast to the previous test